server/graphql/v2/mutation/HostApplicationMutations.ts

```diff
@@ -113,6 +148,24 @@ const HostApplicationMutations = {
         throw new ValidationFailed('This collective application has already been approved');
       }
 
+      // If approving, check if Collective has enough admins
+      const admins = await models.Member.findAll({
+        where: {
+          CollectiveId: account.id,
+          role: MemberRoles.ADMIN,
+        },
+      });
```

Comment:
Should we count MemberInvitations here? I personally think not.

Reply:
We should, at least if we want to be consistent with the specs and designs.